contents/docs/deployment/deploy-source.md

```diff
@@ -43,5 +44,6 @@ If there is no desire to customize your source, we strongly recommend other depl
     ```bash
     ./bin/docker-server & ./bin/docker-worker
     ```
+<br>
 
-    > We reccomend using something like [Supervisor](https://github.com/Supervisor/supervisor) to keep this command running.
+> _We reccomend using something like [Supervisor](https://github.com/Supervisor/supervisor) to keep this command running._
```

Comment:
this line is vague - could we say why this is worth doing or not doing a bit more specifically?